person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [x] Kindly make sure you use the correct syntax while initializing the param of your class. as you may notice how it's impossible to create the instance of your classes in the `irb`
- [x] Also make sure you have an id generator for each instance of your class.
```suggestion
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```

Reply:
@christianbiring1 I appreciate your feedback 👍 But I am afraid `parent_permission = true` will create a linters' error. And the rubocop suggested to use `parent_permission: true` instead. Here is the demonstration from rubocop:
  
![error](https://github.com/emhamza/opp-school-library/assets/122353030/fce1a207-f6a2-402a-9492-5f5543a8c64d)